package.json

```diff
@@ -61,16 +62,20 @@
     "eslint-plugin-unicorn": "^47.0.0",
     "fast-json-stable-stringify": "~2.1.0",
     "fdir": "~6.0.1",
+    "fs-extra": "11.1.1",
     "husky": "^8.0.1",
     "json-schema-to-typescript": "~10.1.5",
+    "klaw": "4.1.0",
     "lint-staged": "^13.0.3",
     "mocha": "~10.2.0",
     "open-cli": "~7.2.0",
     "ora": "~6.3.0",
     "prettier": "~2.8.0",
     "ts-node": "~10.9.1",
     "typescript": "~5.0.2",
+    "ua-parser-js": "1.0.35",
     "web-specs": "^2.41.0",
+    "winston": "3.8.2",
```

Comment:
Let's see if we can avoid this dependency.

Reply:
I replaced the `logger` file with a small shim I added to `update`.